front/cypress/e2e/content_builder/content_builder_navigation.cy.ts

```diff
@@ -75,19 +75,13 @@ describe('Content builder navigation', () => {
 
   it('navigates to live project in a new tab when view project button in content builder is clicked', () => {
     const projectUrl = `/en/projects/${projectSlug}`;
-    cy.visit(`/admin/projects/${projectId}/description`);
 
     cy.intercept('**/content_builder_layouts/project_description/upsert').as(
       'saveContentBuilder'
     );
 
-    cy.get('#e2e-toggle-enable-content-builder')
-      .find('input')
-      .click({ force: true });
-
-    cy.wait('@saveContentBuilder');
-    cy.get('#e2e-content-builder-link').click();
-
+    cy.apiEnableContentBuilder({ projectId });
+    cy.visit(`/admin/content-builder/projects/${projectId}/description`);
     cy.get('#e2e-view-project-button > a')
```

Comment:
Since we test the toggle earlier, just replaced this with a direct visit to the content builder page :) Seemed to be flaky otherwise for some reason.